### PR TITLE
Add JunitXML output format

### PR DESCRIPTION
**Pull request by mirkoboehm**

### CODE REVIEW

**Comment by mirkoboehm** [comment]
Retest this, please

File: ci/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+FROM golang:1.12
+LABEL	maintainer="Mirko Boehm <mboehm@openinventionnetwork.com>"
```
Comment (hemarkus) [inline ci/Dockerfile]: Is this the email address you want to use here?

File: go.mod [hunk 2]
```diff
@@ -3,9 +3,24 @@ module github.com/endocode/shelldoc
...
 require (
+	github.com/coreos/etcd v3.3.13+incompatible // indirect
+	github.com/coreos/go-semver v0.3.0 // indirect
+	github.com/jstemmer/go-junit-report v0.0.0-20190106144839-af01ea7f8024 // indirect
+	github.com/magiconair/properties v1.8.1 // indirect
+	github.com/pelletier/go-toml v1.4.0 // indirect
+	github.com/russross/blackfriday v2.0.0+incompatible // indirect
 	github.com/shurcooL/sanitized_anchor_name v1.0.0 // indirect
-	github.com/spf13/cobra v0.0.3
-	github.com/spf13/pflag v1.0.3 // indirect
```
Comment (hemarkus) [inline go.mod:8]: No pflag anymore. Should be indirect of cobra. Also the sum file still lists pflag.

File: pkg/run/context.go [hunk 3]
```diff
@@ -0,0 +1,65 @@
+package run
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/endocode/shelldoc/pkg/junitxml"
+)
+
+// Context contains the context of an exewcution of the run subcommand.
```
Comment (hemarkus) [inline pkg/run/context.go]: exewcution

File: pkg/run/interactions_test.go [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+)
+
+func TestMain(m *testing.M) {
+	verbose = true
+	failureStops = false
+	os.Exit(m.Run())
+}
+func TestHelloWorld(t *testing.T) {
+	context := Context{
+		// ShellName:    "",
```
Comment (hemarkus) [inline pkg/run/interactions_test.go]: Is this intended? Looks like commented code.

**Review by hemarkus — CHANGES_REQUESTED** [review]
minor clean up but looks good basically

**Pull request merged**